README.md

```diff
@@ -93,7 +93,7 @@ The behaviour of the Azure OpenAI API Simulator is controlled via a range of [Az
 
 ### Extending the Azure OpenAI API Simulator
 
-There are also a number of [Azure OpenAI API Simulator Extension points](./docs/extensions.md) that allow you to customise the behaviour of the Azure OpenAI API Simulator. Extensions can be used to modify the request/response, add latency, or even generate responses.
+There are also a number of [Azure OpenAI API Simulator Extension points](./docs/extending.md) that allow you to customise the behaviour of the Azure OpenAI API Simulator. Extensions can be used to modify the request/response, add latency, or even generate responses.
```

Comment:
Nice spot. I keep making this mistake because I think it should probably be called `extensions` (that would feel more natural to me).

Reply:
I'm happy to rename the file in a future PR 👍